signup-service/src/main/java/io/mosip/signup/services/RegistrationService.java

```diff
@@ -291,7 +292,9 @@ public RegistrationStatusResponse getRegistrationStatus(String transactionId)
         return registrationStatusResponse;
     }
 
-    public JsonNode getSchema() {
+
+    @Cacheable(value = UI_SPEC, key = "'ui_spec'")
```

Comment:
can you please change the key name to "latest"
single quotes are not required @Md-Humair-KK